### ISSUE TITLE
pywin32-225.win32-py2.7.exe & amd64 blocked by Windows Defender SmartScreen?

### ISSUE THREAD

**Issue by kxrob** [mhammond/pywin32#1438]
Trying to install / run these 2:
https://github.com/mhammond/pywin32/releases/download/b225/pywin32-225.win32-py2.7.exe
https://github.com/mhammond/pywin32/releases/download/b225/pywin32-225.win-amd64-py2.7.exe

on up-to-date Win10-64 is immediately blocked by Windows Defender SmartScreen as quoted below*. Though VirusTotal.com is green 0/71 on them.
This does not happen e.g. with:
pywin32-225.win-amd64-py3.7.exe 
pywin32-224.win32-py2.7.exe, 
pywin32-224.win-amd64-py2.7.exe, 
pywin32-220.win-amd64-py2.7.exe, 

Are the 2 blocked versions just to unknow / rarely used for SmartScreen, or is there something odd?

*
```
> Der Computer wurde durch Windows geschützt
> Von Windows Defender SmartScreen wurde der Start einer unbekannten App verhindert. Die Ausführung dieser App stellt u. U. ein Risiko für den PC dar.
> Weitere Informationen
> (The computer was protected by Windows
> Windows Defender SmartScreen prevented the launch of an unknown app. The execution of this app represents u. U. a risk for the PC.
> additional Information)
```

**Comment by Avasam** [comment]
@kxrob @mhammond Given this issue is specifically about Python 2.7 support, which is no longer relevant. Should probably be closed?
